aiocoap/protocol.py

```diff
@@ -241,13 +241,11 @@ async def create_server_context(cls, site, bind=None, *, loggername="coap-server
                     lambda mman: MessageInterfaceSimple6.create_client_transport_endpoint(mman, log=self.log, loop=loop))
             elif transportname == 'tinydtls':
                 from .transports.tinydtls import MessageInterfaceTinyDTLS
-
```

Comment:
Please don't change whitespace or other unrelated code in PRs. If this is your editor's doing, you may want to reign it in on code bases that don't follow that editor's particular coding style.

Reply:
Yep, sorry 'bout that, this one (or two) slipped through. I managed to avoid my urge to run black against the entire codebase and submit that as a PR 😜

Reply:
> The concern is duly noted, and as soon as I have the number of large branches down a bit, and have verified that no code is becoming less readable, black *will* be the one color in which you get aiocoap, and CI will mark any specks of color with a big red cross mark.

The world wants to know- when will we have PEP-484 across the codebase and no red marks next to mypy? 😉

On a serious note though, I'd be happy to help with that effort- for the public interfaces or the entire project- *if* you're interested obviously, and if time permits

Personally I now find it nearly impossible to write good interfaces without it, though that may be evidence of my lack of discipline rather than evidence of the broad value of type annotations